src/atomicui/atoms/List/List.tsx

```diff
@@ -86,44 +99,52 @@ const List: FC<ListProps> = ({
 		import("./styles.scss");
 	}
 
-	const checkIfExternalLink = (link: string): boolean =>
-		!!(link?.startsWith("https://") || link?.startsWith("http://") || link?.startsWith("#"));
-
 	return (
 		<View data-testid="list-container" as="ul" className={ulClass} {...omit(["className"], props)}>
-			{listArray.map(item => {
-				const isExternalLink = checkIfExternalLink(item?.link || "#");
-				const hasSubMenuItems = !!item.subMenu?.length;
+			{listArray.map(({ link, subMenu, label, iconContainerClass, iconBeforeLink, isExternalLink }) => {
+				const hasSubMenuItems = !!subMenu?.length;
 
 				return (
-					<li data-testid={item.label} key={uuid.randomUUID()} className="list-item">
+					<li
+						data-testid={label}
+						key={uuid.randomUUID()}
+						className={SHOW_NEW_NAVIGATION ? "new-list-item" : "list-item"}
+						onMouseEnter={e => hasSubMenuItems && setPos(subMenu.length > 5 ? e.clientY - 100 : e.clientY - 50)}
```

Comment:
This wont have to be updated if the styles change in future, this is only done to take care of last menu item "Industry" that has more than 5 sub menu items, and gets cut off from the bottom since it's placed in the last.